### PR TITLE
Introduce @lvksh/logger

### PR DESCRIPTION
**Pull request by lucemans**

### CODE REVIEW

**Comment by lucemans** [comment]
As gorgeous as it could be
image

**Comment by TheOtterlord** [comment]
We have decided not to use this library and politely ask for you to respect this decision.

**Review by svemat01 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -21,7 +22,7 @@
     "@types/moment": "^2.13.0",
     "@types/node": "^15.6.0",
     "axios": "^0.21.1",
-    "chalk": "^4.1.1",
+    "chalk": "4",
```
Comment (HonbraDev) [inline package.json]: yes

File: src/classes/LGRBase.ts [hunk 2]
```diff
@@ -1,63 +1,13 @@
 import chalk from "chalk";
-import moment from "moment";
+import { MethodConfig } from "@lvksh/logger";
```
Comment (HonbraDev) [inline src/classes/LGRBase.ts:2]: even more yes

File: src/classes/Logger.ts [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+const all = createLogger({
+  base: chalk.yellow("[Server]"),
+  discord: chalk.hex("#7289DA")("[Discord]")
+});
 
 export class Logger {
-  public static base = new LGRBase();
-  public static discord = new LGRBase(chalk.hex("#7289DA")("[Discord]"));
+  public static base = createLogger(LGRBase, {}, all.base);
+  public static discord = createLogger(LGRBase, {}, all.discord);
```
Comment (HonbraDev) [inline src/classes/Logger.ts:12]: very very yes

**Review by HonbraDev — COMMENTED** [review]
yes

**Pull request closed without merging**